.github/workflows/cli-docs-update.yml

```diff
@@ -0,0 +1,33 @@
+name: CLI commands docs check
+
+on:
+  pull_request:
+    branches:
+      - master
+      - support/**
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    name: cli-docs-check
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Save current CLI docs files
```

Comment:
can we just try to generate docs and see if `git diff`/\*smth else\* says something new was done?